### PR TITLE
System messages for new feed items in the space #1950

### LINKED ISSUE #1950: System messages for new feed items in the space
A user create a new feed item to this common (space) -> message: "@userName created the $feedItemType $feedItemTitle." where the $feedItemTitle should be a link to the feedItem and $feedItemType should be discussion\proposal\channel according to type of this feed item.
Notice that we should filter out items which aren't visible in the feed, such as Membership request proposal. Also hide items with Visibility higher than the base circle.

### PR DESCRIPTION
**Pull request by ariel-dao**
Added system message definition that's executed on new feed item creation.
Common member added and other stuff that don't create feed items should be covered.
There are a few checks before creating: Titles and circle visibility.

### CODE REVIEW

**Comment by ariel-dao** [comment]
@danielr18 @andreymikhadyuk Is there a known way to send text as a link in rich text?

File: functions/src/commons/business/createSystemMessagesOnFeedItemCreated.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  FeedObjectTypes,
+  SystemDiscussionMessageType,
+} from "../../shared/interfaces";
+import { userDb } from "../../users/db";
+import { commonsDb } from "../db/common";
+import { commonFeedDb } from "../db/commonFeed";
+import { proposalsDb } from "../../proposals/db/proposals";
+import { governanceDb } from "../../governance/db";
+
+export const createSystemMessagesOnFeedItemCreated = async (
```
Comment (danielr18) [inline functions/src/commons/business/createSystemMessagesOnFeedItemCreated.ts:16]: Where will this function be called?

**Review by danielr18 — CHANGES_REQUESTED** [review]

File: functions/src/commons/business/createSystemMessagesOnFeedItemCreated.ts (hunk 1 above)
Comment (ariel-dao) [reply functions/src/commons/business/createSystemMessagesOnFeedItemCreated.ts:16]: Missed a commit, fixed

**Comment by elatif2020** [comment]
> @danielr18 @andreymikhadyuk Is there a known way to send text as a link in rich text?

@ariel-dao 
we do it for the system messages about new space created

**Comment by andreymikhadyuk** [comment]
@elatif2020 @ariel-dao @danielr18 we [construct](https://github.com/daostack/common-web/blob/dev/src/shared/components/Chat/ChatMessage/utils/getTextFromSystemMessage.tsx#L130) system messages on the FE side based on the type and data from BE.
There are also last messages which use just text from BE, but they are not clickable, so that info may contain just plain text.

**Review by danielr18 — APPROVED** [review]

**Pull request merged**